samples/MvvmSampleUwp/App.xaml.cs

```diff
@@ -39,14 +41,24 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
             TitleBarHelper.StyleTitleBar();
             TitleBarHelper.ExpandViewIntoTitleBar();
 
-            // Register services
-            Ioc.Default.ConfigureServices(
-                new ServiceCollection()
-                .AddSingleton<IFilesService, FilesService>()
-                .AddSingleton<ISettingsService, SettingsService>()
-                .AddSingleton(RestService.For<IRedditService>("https://www.reddit.com/"))
-                .BuildServiceProvider());
-        }
+                // Register services
+                Ioc.Default.ConfigureServices(
```

Comment:
Just noticed: looks like this accidentally got indented 4 spaces too much?
Same for the closing bracket at the end 🤔

Reply:
merge issue ) fixed